design.css

```diff
@@ -1,24 +1,54 @@
 body {
-  background-color: #8EC5E7;
   background-image: url(img/cold.pattern.jpg);
   background-size: 40%;
   background-repeat: repeat;
 
 }
 
 body.tropic {
-  background-color: #8F2F28;
+
   background-image: url(img/africa.pattern.jpg);
   background-size: 80%;
 
 
 }
 
+section
+{
+  background-color: rgb(76, 92, 185);  
+}
+
+section.tropic
+{
+  background-color: rgb(69, 189, 179);
+  color: #000;
+}
+
+/* Textfärger separat för att inte paja formattering i dropdown */
+section>div{
+  color: white;
+}
+section.tropic>div{
+  color: black;
+}
+
+
+
 header {
   display: flex;
   justify-content: end;
 }
 
+.description{
```

Comment:
Why is it two of the same? .description?

Reply:
Most likely a mistake in the merge... Fixed in new commit